src/Illuminate/Support/Number.php

```diff
@@ -17,6 +18,136 @@ class Number
      */
     protected static $locale = 'en';
 
+    /**
+     * Get a new numeral instance for the given value.
+     *
+     * @param  int|float  $value
+     */
+    public static function of($value): Numeral
+    {
+        return new Numeral($value);
+    }
+
+    /**
+     * Determine if the given value is a number.
+     *
+     * @param  mixed  $number
+     */
+    public static function isNumeric($number): bool
+    {
+        return is_numeric($number);
+    }
+
+    /**
+     * Determine if the given value is an even number.
+     *
+     * @param  mixed  $value
+     */
+    public static function isEven($value): bool
+    {
+        return $value % 2 === 0;
+    }
+
+    /**
+     * Determine if the given value is an odd number.
+     *
+     * @param  mixed  $value
+     */
+    public static function isOdd($value): bool
+    {
+        return $value % 2 !== 0;
+    }
+
+    /**
+     * Determine if the given value is a float.
+     *
+     * @param  mixed  $value
+     */
+    public static function isFloat($value): bool
+    {
+        return self::isNumeric($value) && is_float($value + 0);
```

Comment:
Why `+ 0`?

Reply:
A numeral string can be passed in ( "11.0" for instance), which will result in `true` after the first check, but `false` in the `is_float` if I'm not mistaken. Adding the zero as an integer will cast it to avoid this false negative.

However, should we consider a numeral string as a number? What are your thoughts?